platform.txt

```diff
@@ -70,24 +70,24 @@ compiler.path={runtime.tools.xtensa-lx106-elf-gcc.path}/bin/
 compiler.sdk.path={runtime.platform.path}/tools/sdk
 
 compiler.libc.path={runtime.platform.path}/tools/sdk/libc/xtensa-lx106-elf
-compiler.cpreprocessor.flags=-D__ets__ -DICACHE_FLASH -U__STRICT_ANSI__ -D_GNU_SOURCE -DESP8266 {build.opt.flags} "-I{compiler.sdk.path}/include" "-I{compiler.sdk.path}/{build.lwip_include}" "-I{compiler.libc.path}/include" "-I{build.path}/core"
+compiler.cpreprocessor.flags=-D__ets__ -DICACHE_FLASH -U__STRICT_ANSI__ -D_GNU_SOURCE -DESP8266 -Os {build.opt.flags} "-I{compiler.sdk.path}/include" "-I{compiler.sdk.path}/{build.lwip_include}" "-I{compiler.libc.path}/include" "-I{build.path}/core"
```

Comment:
@d-a-v I would like to suggest a slight modification. Moving/consolidating the three `-Os` to here permits override from `build.opt.flags` as well as `build.debug_port. From what I have read, GCC uses the last optimization specified.

With the above change, I would leave `( '.menu.dbg.Disabled.build.debug_port', '' )` as is so that `build.opt` would have more control for defining global optimization in the absence of the `Debug port` specification.